pkg/models/convert.go

```diff
@@ -0,0 +1,60 @@
+package models
+
+import "github.com/solo-io/squash/pkg/api/v1"
+
+func ConvertDebugAttachments(das []*v1.DebugAttachment) []DebugAttachment {
+	out := []DebugAttachment{}
+	for _, da := range das {
+		out = append(out, ConvertDebugAttachment(da))
+	}
+	return out
+}
+
+func ConvertDebugAttachment(da *v1.DebugAttachment) DebugAttachment {
```

Comment:
where is the convert used? is this for the REST api that the envoy filter uses?

Reply:
this is needed for compatibility with the extensions until we update them